core/src/display_object.rs

```diff
@@ -1498,6 +1498,9 @@ pub trait TDisplayObject<'gc>:
     fn name(&self) -> AvmString<'gc> {
```

Comment:
It might be cleaner to return `Option<AvmString<'gc>>` here instead.

Reply:
I was about to do this, then realized that there are several more places where `name` is used with a default of an empty string. I don't know which of those match Flash and which don't, so I decided to keep this default here.